src/Generating/StructGen.h

```diff
@@ -16,7 +16,7 @@
 #include "ComposableGenerator.h"
 #include "../Noise/Noise.h"
 
-
+#include <functional>
```

Comment:
This doesn't seem to be used by the header, move it to cpp file, if it is needed at all.

Reply:
it is no more used i removed it